changelog/unreleased/kong/fix-ip-restriction-tcp-error.yml

```
@@ -0,0 +1,5 @@
+message: |
```

Actually I want the newlines here to be replaced with spaces, so I should use `>` instead of `|`. See: https://yaml-multiline.info/

```suggestion
message: >
```